clang/lib/Sema/SemaOverload.cpp

```diff
@@ -123,82 +123,59 @@ CompareDerivedToBaseConversions(Sema &S, SourceLocation Loc,
 /// GetConversionRank - Retrieve the implicit conversion rank
 /// corresponding to the given implicit conversion kind.
 ImplicitConversionRank clang::GetConversionRank(ImplicitConversionKind Kind) {
-  static const ImplicitConversionRank
-    Rank[] = {
-    ICR_Exact_Match,
-    ICR_Exact_Match,
-    ICR_Exact_Match,
-    ICR_Exact_Match,
-    ICR_Exact_Match,
-    ICR_Exact_Match,
-    ICR_Promotion,
-    ICR_Promotion,
-    ICR_Promotion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_OCL_Scalar_Widening,
-    ICR_Complex_Real_Conversion,
-    ICR_Conversion,
-    ICR_Conversion,
-    ICR_Writeback_Conversion,
-    ICR_Exact_Match, // NOTE(gbiv): This may not be completely right --
-                     // it was omitted by the patch that added
-                     // ICK_Zero_Event_Conversion
-    ICR_Exact_Match, // NOTE(ctopper): This may not be completely right --
-                     // it was omitted by the patch that added
-                     // ICK_Zero_Queue_Conversion
-    ICR_C_Conversion,
-    ICR_C_Conversion_Extension
-  };
+  static const ImplicitConversionRank Rank[] = {
```

Comment:
I am pretty sure clang-format asked you to make this impossible to read; can you ignore clang-format here and leave it in the previous form?

Reply:
How do you feel about me wrapping this in `// clang-format off ... // clang-format on` which will both keep the automation happy and prevent this in the future?

Reply:
I don't know that we have a policy one way or the other, but my personal preference is to not use those comments unless the file is already fully formatted -- they're too distracting and we have too much unformatted code to want to slap them all over the code base. But others may feel differently!

Reply:
By fixing the intending (but keeping one per line) I managed to both make clang-format happy and keep roughly the same formatting. Is that good with you?